app/src/componentLibrary/attributeList/editableAttribute/attribute/attribute.jsx

```diff
@@ -37,16 +38,28 @@ export const Attribute = ({
   handleCrossIconKeyDown,
   wrapperRefCallback,
 }) => {
+  const [isLabelFocused, setIsLabelFocused] = useState(false);
```

Comment:
Looks like now, the naming does not match the expected usage.
It is not only about label.

Reply:
Done.